apps/console/src/public/deployment.config.json

```diff
@@ -1773,6 +1776,7 @@
         "routes": {
             "organizationEnabledRoutes": [
                 "accountSecurity",
+                "actions",
```

Comment:
this will enable the actions UI in suborgs, is it expected?

Reply:
Yes. This is expected.